openstack/loadbalancer/v2/apiversions/requests.go

```diff
@@ -0,0 +1,13 @@
+package apiversions
+
+import (
+	"github.com/gophercloud/gophercloud"
+	"github.com/gophercloud/gophercloud/pagination"
+)
+
+// lists all the load balancer API versions available to end-users.
```

Comment:
This comment should start with `List` so it will look good on godoc.org:
`// List lists all the API versions available to end-users.`